README.md

```diff
@@ -1,157 +1,179 @@
+| [Rails](#rails)                 | rails-instrumentation                  | >= 3.0.0           |
+| [Redis](#redis)                 | redis-instrumentation                  | >= 4.0.0           |
+| [RestClient](#restclient)       | restclient-instrumentation             | >= 1.5.0           |
+| [Sequel](#sequel)               | sequel-instrumentation                 | >= 3.47.0          |
+| [Sidekiq](#sidekiq)             | sfx-sidekiq-opentracing                | >= 0.7.0           |
+| [Sinatra](#sinatra)             | sinatra-instrumentation                | >= 1.0.0           |
+
+## Configuration values
+
+| Configure parameter | Environment variable  | Default                          | Notes |
+| ------------------- | --------------------  | -------------------------------- | ----- |
+| tracer              | N/A                   | `nil`                            | The OpenTracing global tracer. |
+| ingest_url          | SIGNALFX_ENDPOINT_URL | `http://localhost:9080/v1/trace` | The endpoint the tracer sends spans to. Send spans to a Smart Agent, OpenTelemetry Collector, or a SignalFx ingest endpoint. |
+| service_name        | SIGNALFX_SERVICE_NAME | `signalfx-ruby-tracing`          | The name to identify the service in SignalFx. |
+| access_token        | SIGNALFX_ACCESS_TOKEN | `''`                             | The SignalFx organization access token. |
+
+## Install the SignalFx Tracing Library for Ruby
+
+Follow these steps to install the tracing library. You can either use the
+bootstrap utility to install the tracing library and its dependencies or
+manually install everything. 
+
+The bootstrap utility updates your Gemfile and installs the required
+dependencies for you. For information about the bootstrap utility, see the
+[sfx-rb-trace-bootstrap](bin/sfx-rb-trace-bootstrap) file.
+
+The steps assume you have RubyGems and Bundler.
+
+### Install the library with the bootstrap utility
+
+1. Set the service name to identify the service in SignalFx:
+   ```bash
+   $ export SIGNALFX_SERVICE_NAME = "yourService"
+   ```
+2. If you're sending trace data to a SignalFx ingest endpoint or OpenTelemetry
+   Collector, set the access token for your organization:
+   ```bash
+   $ export SIGNALFX_ACCESS_TOKEN = "yourToken"
+   ```
+3. If you're sending trace data to an endpoint that's not `http://localhost:9080/v1/trace`, 
+   set the endpoint:
+   ```bash
+   $ export SIGNALFX_ENDPOINT_URL = "yourEndpoint"
+   ```
+4. Download the [latest release](https://github.com/signalfx/signalfx-ruby-tracing/releases/latest) of the tracing library.
+5. Install the tracing library:
+   ```bash
+   $ gem install signalfx-tracing
+   ```
+6. View the list of instrumentations you can install with the bootstrap utility:
+   ```bash
+   $ sfx-rb-trace-bootstrap --list
+   ```
+7. Use the bootstrap utility to install applicable instrumentations for your
+   application:
+   ```bash
+   $ sfx-rb-trace-bootstrap --install-deps <dep1>,<dep2>,<dep3>,<dep4>
```

Comment:
I'd show an example of this so it's clear which items to use from the `--list` output.